pysmt/solvers/bdd.py

```diff
@@ -87,6 +148,22 @@ def __init__(self, environment, logic, user_options):
         self.backtrack = []
         self.latest_model = None
 
+    def get_default_options(self, logic=None, user_options=None):
+        res = BddOptions()
+        if user_options is not None:
+            if "unsat_cores_mode" in user_options:
```

Comment:
So, the proper way to create this is {"unsat_cores_mode": None}? Is this because of how the factory works? Wouldn't it be better just to assert that "unsat_cores_mode" not in user_options?